calicoctl/calico_ctl/node.py

```diff
@@ -574,6 +575,9 @@ def node_stop(force):
     Stop the Calico node.  This stops the containers (calico/node and
     calico/node-libnetwork) that are started by calicoctl node.
     """
+    # The command has to be run as root to stop the calico-node service
```

Comment:
Long story short, yes we do want the `systemctl` commands here because that is how we manage the `calico/node` container when running in rkt.